test/tests/ide/jetbrains/gateway_test.go

```diff
@@ -251,11 +251,11 @@ func TestGoLand(t *testing.T) {
 	f := features.New("Start a workspace using GoLand").
 		WithLabel("component", "IDE").
 		WithLabel("ide", "GoLand").
-		Assess("it can let JetBrains Gateway connect", func(_ context.Context, t *testing.T, cfg *envconf.Config) context.Context {
+		Assess("it can let JetBrains Gateway connect", func(testCtx context.Context, t *testing.T, cfg *envconf.Config) context.Context {
 			ctx, cancel := context.WithTimeout(context.Background(), 30*time.Minute)
```

Comment:
should not we pass testCtx here as an argument to context.WithTimeout

Reply:
Could you review all other files please, it seems to be a common problem.

Reply:
I guess that's possible now that the test context shouldn't be cancelled. I'll update all tests.

Reply:
Done